doc/source/project/releases.rst

```diff
@@ -0,0 +1,70 @@
+======================
+Galaxy Release Process
+======================
+
+This document outlines the Galaxy release cycle so administrators and users know
+when to expect updates and what each update includes, making it easier to plan
+for upgrades and maintain their systems.  It does not cover the actual execution
+of the release process, which is covered elsewhere. (link)
+
+
+Major or Long Term Support (LTS) Releases
+-----------------------------------------
+
+The first release of each year is Galaxy's annual LTS release. The LTS release
+is a major version update corresponding to the year, for example, 24.0.  This
+release receives significant and thorough testing by a dedicated team that does
+end-to-end testing of a variety of analysis protocols and will be supported and
+receive bugfixes until the next LTS.  As such, this release is ideally targeted
+for production use by local Galaxy administrators.
+
+
+Minor Releases
+--------------
+
+Throughout the year, we also ship several (currently 2-3) minor releases. These
```

Comment:
```suggestion
Throughout the year, we also ship several minor releases. These
```
or it'll tie us to 2-3

Reply:
I don't think this ties us to 2-3 at all, just sets expectations by communicating the current frequency?

Reply:
"usually" ?